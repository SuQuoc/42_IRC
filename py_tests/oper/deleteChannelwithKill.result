:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 005 client0 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:AfterLife 001 client1 :Welcome to the Internet Relay Network, user1
:AfterLife 005 client1 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:client1!user1@10.12.2.4 JOIN #chan * :user1
:AfterLife 442 client0 #chan :You're not on that channel
:AfterLife 491 client0 :No O-lines for your host
:AfterLife 481 client0 :Permission Denied- You're not an IRC operator
:AfterLife 442 client0 #chan :You're not on that channel
