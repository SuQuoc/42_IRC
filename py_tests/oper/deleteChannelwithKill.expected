:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 005 client0 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:AfterLife 001 client1 :Welcome to the Internet Relay Network, user1
:AfterLife 005 client1 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:client1!user1@10.12.2.4 JOIN #chan * :user1
:AfterLife 442 client0 #chan :You're not on that channel
:AfterLife 381 client0 :You are now an IRC operator
:client0!user0@10.12.2.4 KILL client1 :kill client1
:AfterLife 403 client0 #chan :No such channel
